translate/storage/properties.py

```diff
@@ -886,11 +886,11 @@ def __str__(self):
     def getoutput(self):
         """Convert the element back into formatted lines for a .properties file"""
         notes = self.getnotes()
-        if notes:
-            notes += "\n"
         if self.isblank():
-            return notes + "\n"
+            return notes + '\n'
```

Comment:
Please keep existing coding style, we're using [black](https://github.com/psf/black) code formatter to enforce coding style.

```suggestion
            return notes + "\n"
```